tracetools/src/tracetools.c

```diff
@@ -489,6 +491,17 @@ DEFINE_TRACEPOINT(
   TRACEPOINT_ARGS(
     buffer))
 
+void __attribute__((constructor)) tracetools_init()
```

Comment:
We probably don't _need_ a shared lib destructor, but it might be a good idea: looks like lttng-ust has things to clean up https://github.com/lttng/lttng-ust/blob/83d51a7498a113adc2c124e502edfa717564f630/src/lib/lttng-ust/lttng-ust-comm.c#L2397-L2400

Reply:
Added a destructor in 814bdd421e14cb4a6a469719edbe1e39f96fab99